components/raftstore/src/store/fsm/apply.rs

```diff
@@ -3432,6 +3499,19 @@ where
                 None => break,
             }
         }
+        let s = self.delegate.yield_state.as_mut().map_or(0, |s| {
+            if s.approximate_size == 0 {
+                s.approximate_size = s.heap_size();
+            }
+            s.approximate_size
+        });
+        let trace = ApplyMemoryTrace {
+            pending_cmds: self.delegate.pending_cmds.heap_size(),
```

Comment:
Don’t need to track `pending_msgs` and `pending_entries`?

Reply:
They are included in `rest_size`